### DISCUSSION TITLE
Is there a way to see old versions docs

### DISCUSSION THREAD

**Discussion by saschahofmann** [ibis-project/ibis#7520] [Q&A] [answered]
Some projects enable you to change the release version of the docs you're looking at. Any chance I am just not finding this or is that currently not supported? Would be super useful especially since you guys move so rapidly !

**Comment by gforsyth** [comment]
Hey @saschahofmann -- it's not currently supported. We had versioned docs for a little while but it was a pain to maintain.

That said, we could probably render the docs for old versions as a one off and then publish them to a subdirectory on the website. 
I can try playing around with that next week.

**Reply by saschahofmann** [reply]
Yeah I can see how trying to integrate old docs into the same site could be a pain.
I think everything that makes it easier than trying to read through the source code of an old version is already helpful!

**Reply by gforsyth** [reply, marked as answer]
Hey @saschahofmann !  I've pushed up old versions of the ibis docs to our `gh-pages` branch.  The artisanal index.html file isn't available yet, but you can go to https://ibis-project.github.io/ibis/4.0.0/ for the rendered docs for 4.0.0, and all of our released tags between 4.0.0 and 6.2.0 are available that way.

**Reply by saschahofmann** [reply]
Amazing thank you ! Will check them out
